File: fizzbuzz/Perl/fizzbuzz.pl
#!/usr/bin/env perl
# Use with perl fizzbuzz.pl
# Classic fizzbuzz problem from 1 to 100, in this case using ternary operator.

use strict;
use warnings;


for my $i (1..100){

    print $i % 3 ? "":"Fizz";
    print $i % 5 ? "":"Buzz";
    print $i % 3 && $i % 5 ? $i:"";
    print("\n");
    
}

File: fizzbuzz/JavaScript/fizzbuzz.js
// Use js fizzbuzz.js to run
// Written for NodeJs
// Classic fizzbuzz problem from 1 to 100

for(let i = 1; i <= 100; i++){
    if(i % 3 === 0 && i % 5 === 0){
        console.log('FizzBuzz');
    }
    else if(i % 3 === 0){
        console.log('Fizz');
    }
    else if(i % 5 === 0){
        console.log('Buzz');
    }
    else{
        console.log(i);
    }
}
